src/EditorFeatures/CSharpTest/Diagnostics/UseImplicitOrExplicitType/UseExplicitTypeTests.cs

```diff
@@ -1186,5 +1186,31 @@ static void M()
 }",
 options: ExplicitTypeEverywhere());
         }
+
+        [WpfFact, Trait(Traits.Feature, Traits.Features.CodeActionsUseExplicitType)]
```

Comment:
These tests likely shouldn't be WpfFacts. @dpoeschl @jasonmalinowski do you know why these would need that.  

@zaytsev-victor (you don't need to change this).